tsl/src/continuous_aggs/materialize.c

```diff
@@ -747,12 +749,25 @@ insert_materialization_invalidation_logs(List *caggs, List *invalidations,
 	{
 		ContinuousAgg *ca = lfirst(lc);
 		int32 cagg_id = ca->data.mat_hypertable_id;
+		int64 ignore_invalidation_older_than = ca->data.ignore_invalidation_older_than;
 		foreach (lc2, invalidations)
 		{
 			Invalidation *entry = (Invalidation *) lfirst(lc2);
+			int64 minimum_invalidation_time =
```

Comment:
could you add an example at the top of this function to describe/clarify how min invalidation time is used to ignore some invalidations.